Migrating playbooks and roles to Core 2.12 (#313)
RedHatInsights/red-hat-ansible-automation-platform-documentation: downstream/modules/platform/proc-migrate-playbooks-roles.adoc

[id="proc-migrate-playbooks-roles_{context}"]

= Migrating your Ansible playbooks and roles to Core 2.12

[role="_abstract"]

> When you are migrating from non collection-based content to collection-based content, you should use Fully Qualified Collection Names (FQCN) when using Ansible conentt collections in playbooks and roles. If the collection name is not defined, you may encounter behavioral changes within your environment.

Reviewer: ```suggestion
When you are migrating from non collection-based content to collection-based content, you should use Fully Qualified Collection Names (FQCN) when using Ansible content collections in playbooks and roles. If the collection name is not defined, you may encounter behavioral changes within your environment.
```

> When you are migrating from non collection-based content to collection-based content, you should use Fully Qualified Collection Names (FQCN) when using Ansible conentt collections in playbooks and roles. If the collection name is not defined, you may encounter behavioral changes within your environment.

Author: Not at all. Thanks for the peer-review 👍🏽 
BTW, do you see anything on your end that may be keeping the "commit suggestion" button to be disabled? I can't find anything on my end.

> When you are migrating from non collection-based content to collection-based content, you should use Fully Qualified Collection Names (FQCN) when using Ansible content collections in playbooks and roles. If the collection name is not defined, you may encounter behavioral changes within your environment.

Author: ```suggestion
When you are migrating from non collection-based content to collection-based content, you should use the Fully Qualified Collection Names (FQCN) in playbooks and roles to avoid unexpected behavior.
```

> When you are migrating from non collection-based content to collection-based content, you should use Fully Qualified Collection Names (FQCN) when using Ansible conentt collections in playbooks and roles. If the collection name is not defined, you may encounter behavioral changes within your environment.

Author: Yes, it said that the suggestion was outdated so I just redid what you suggested and it worked. Great team work!
